src/ui/components/Library/Team/View/Tests/hooks/useTests.ts

```diff
@@ -0,0 +1,25 @@
+import { useContext } from "react";
+import { useImperativeCacheValue } from "suspense";
+
+import { GraphQLClientContext } from "replay-next/src/contexts/GraphQLClientContext";
+import { TeamContext } from "ui/components/Library/Team/TeamContextRoot";
+import useToken from "ui/utils/useToken";
+
+import { testsCache } from "../../TestRuns/suspense/TestsCache";
+
+const EMPTY_ARRAY: any[] = [];
+
+export function useTests(): any[] {
```

Comment:
Replace `any[]` with an actual type.

`useTestRuns()` returns a `TestRun[]`. This should probably return a `Test[]` array (presumably the `Test` type that you added to `packages/shared/test-suites/TestRun`)